Classes/Service/MigrateRelationsService.php

```diff
@@ -259,6 +324,7 @@ protected function execSelectDamReferencesWhereSysFileExists() {
 			sys_file_metadata.title,
 			sys_file_metadata.description,
 			sys_file_metadata.alternative,
+			sys_file_metadata.caption,
```

Comment:
This breaks a lot migrations as the caption field is added by EXT:filemetadata which is deactivated by default and a SQL error is thrown now.